app_power_module/src/telemetry.c

```diff
@@ -112,16 +127,25 @@ void adc_task(void) {
 
     while (true) {
         k_timer_status_sync(&adc_task_timer);
-        gpio_pin_toggle_dt(&led);
 
         if (0 < l_read_adc_mv(&vin_sense_adc, &vin_sense_sequence, (int32_t *) &vin_adc_data_mv)) {
             LOG_ERR("Failed to read ADC value from %d", vin_sense_adc.channel_id);
             continue;
         }
 
         float vin_adc_data_v = (vin_adc_data_mv * mv_to_v_multiplier) * adc_gain;
-        if (k_msgq_put(&ina_telemetry_msgq, &vin_adc_data_v, K_NO_WAIT)) {
-            LOG_ERR("Failed to put data into ADC processing queue");
+        if (k_msgq_put(&adc_telemetry_msgq, &vin_adc_data_v, K_NO_WAIT)) {
+        }
+
+        // Buffer up data for logging before boost. If no space, throw out the oldest entry.
+        if (!logging_enabled && k_msgq_num_free_get(&adc_logging_msgq) == 0) {
+            float throwaway_data;
+            k_msgq_get(&adc_logging_msgq, &throwaway_data, K_NO_WAIT);
         }
+
+        if (k_msgq_put(&adc_logging_msgq, &vin_adc_data_v, K_MSEC(ADC_UPDATE_TIME_MS))) {
+        }
```

Comment:
Theres nothing here is that on purpose?

Reply:
Yea originally debating on keeping a log statement in there or not

Reply:
Will just wipe for now